doc/source/ray-core/actors/fault-tolerance.rst

```diff
@@ -14,6 +14,10 @@ After the specified number of restarts, subsequent actor methods will
 raise a ``RayActorError``.
 You can experiment with this behavior by running the following code.
 
+.. note:: 
```

Comment:
This is true for not just `__init__` but any other method of an actor.

Reply:
Hmmm, maybe the ambiguity for `__init__` is how it differs from the normal actor method. It won't get retry for example.

Reply:
Yes it's can't retry b/c of RayActorError

Reply:
Actually, I think it might still be better to keep it since when an actor restarts, only the `__init__` function is relevant. 

One alternative scenario that this rules out is "ray retrying to restart even if __init__ function fails".

Reply:
hmm should we add a unit test to verify the behavior? i.e. 
constructor throws exception -> no retry
constructor failed without exception -> no retry
other function throws exception -> retry
other function failed without exception -?

Reply:
Sure - we could do that.